assets/i18n/en.json

```diff
@@ -345,7 +345,7 @@
     "I18N_LIBRARY_PAGE_TITLE": "Community Library Lessons | Oppia",
     "I18N_LIBRARY_RATINGS_TOOLTIP": "Ratings",
     "I18N_LIBRARY_SEARCH_PLACEHOLDER": "What are you curious about?",
-    "I18N_LIBRARY_SUB_HEADER": "Begin your adventure by browsing our explorations.",
+    "I18N_LIBRARY_SUB_HEADER": "Browse the full set of lessons created by the community",
```

Comment:
Question: wouldn't we have to change the translation in other files as well?

Reply:
Other languages will be updated automatically once they have been translated in translatewiki.net.

Reply:
Ah ok!